library.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "printf",
+  "version": "0.6.3",
```

Comment:
I don't suppose it's possible to avoid duplicating the version number here? ...

Reply:
I'm not sure what you mean by this.